doc/guidelines/frontend-ui-usability.md

```diff
@@ -0,0 +1,195 @@
+# Frontend UI & Usability
+
+## Terminology
+
+### Usability
+
+> Extent to which a system, product or service can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use.
+
+### User Interface (UI)
+
+> Set of all the components of an interactive system that provide information and controls for the user to accomplish specific tasks with the interactive system.
+
+### User Experience (UX)
+
+> Combination of user's perceptions and responses that result from the use and/or anticipated use of a system, product or service.
+
+## Guidelines
+
+This guide should help improve the UI (User Interface) and usability of the PermaplanT web application going forward.  
+The following guidelines are:
+
+- on the one hand a mix of general UI and usability best practices and
+- on the other hand design suggestions tailored to PermaplanT.
+
+### Text
+
+- **Size**
+
+  - `<h1>` Headings : `text-2xl`
+  - `<h2>` Subheadings: `text-xl`
+  - `<h3>` Subheadings (currently not used/needed): `text-lg`
+  - Navigation items: `text-lg`
+  - Running text: `text-sm`
+  - Form fields and labels: `text-sm`
+
+- **Family**
+
+  - PermaplanT is using Tailwind's default cross-browser sans-serif stack everywhere.
+  - If a different family is to be used in the future, extend the theme with a new `theme.fontFamily` section in our Tailwind config file (`tailwind.config.js`).
+
+- **Alignment**
+
+  - Always left align everything.
+  - Exceptions:
+    - Modals: center headings
+    - Guided tour: center highlighted actions
+
+- **Spacing / margin**
+
+  - `<h1>` Headings: `mb-8`
+  - `<h2>` Subheadings: `mb-2`
+  - Cards (like on our Map overview): `mb-4`
+  - Form fields (like on the Seeds page): `mb-8`
+  - Labels of form fields: `mb-2`
+
+- **Letter spacing**
+
+  - Always go with the default value, i.e. `tracking-normal`.
+  - In case letter spacing changes are considered in the future:
+    - Headings: use negative values for the bigger headings h1-h3, i.e. `tracking-tighter` or `tracking-tight`
+    - Running text: `tracking-wide` or `tracking-wider`
+
+### Colors
```

Comment:
Which color should be used for cancel buttons e.g (cancel tour, cancel map/seed creation)?